docs/_partners/faq.md

```diff
@@ -7,136 +7,63 @@ redirect_from: /partners/
+Hopefully, you have a bunch of keen users waiting to get their hands on your Zapier integration; don't be afraid to reach out to them for early testing. Active usage gives us confidence your integration is useable and functionally valuable for numerous users. Make sure to test the integration early and often in the Zap Editor yourself!
 
-## Check Your App Integration Style
+## Maintain Consistency in Your Integration Style
 
-Users should already be familiar with the terminology used in your Zapier integration—it should be consistent with what is used in your app's own UI. It should also be consistent with the style used in other popular Zapier integrations.
+Users should be familiar with the terminology used in your Zapier integration. The input and output field labels, and trigger/action/search names must be consistent with the terminology used in your platform's own UI. It should also be consistent with the style used in other popular Zapier integrations.
 
-We strive to give users a consistent experience throughout Zapier. Be sure to follow [Zapier's standards for app branding](https://platform.zapier.com/partners/planning-guide#how-to-brand-your-zapier-integration) to make your app fit into the Zapier ecosystem well.
+We strive to give users a consistent experience throughout Zapier. Be sure to follow [Zapier's standards for app branding](https://platform.zapier.com/partners/planning-guide#how-to-brand-your-zapier-integration) so your integration fits well into the Zapier ecosystem.
 
 ## Make Connecting New Accounts Easy
 
-Connecting to Zapier must be easy for the user. OAuth v2 is the preferred authentication scheme; otherwise, make sure users can easily retrieve or generate an API Key or other credentials themselves without needing support assistance. Also, the Test Trigger should provide actionable error messages to help users successfully connect their account.
-
-## Provide a Valid Test Account
-
-The App Review Team will go through each Trigger, Action, or Search in your integration to verify that it’s working in a similar way the interactions should work within your app. In order to do this, set up a valid test account and share the credentials with our team (ideally with contact@zapier.com as the account email address). Make sure the account includes all the features needed to test out each Zap step, with any special configurations set. If features require an environment that is hard to replicate or require specific hardware, provide a demo video or documentation to the Zapier team.
+Connecting to Zapier must be easy for the user. OAuth v2 is the preferred authentication scheme; otherwise, users must be able to easily retrieve or generate an API Key or other credentials themselves without needing support. Both authentication and test authentication requests should return actionable, user-friendly error messages to guide users in successfully connecting their account.
```

Comment:
```suggestion
Connecting to Zapier must be easy for the user. OAuth v2 is the preferred authentication scheme; otherwise, users must be able to easily retrieve or generate an API Key or other credentials themselves without needing support. Both authentication and test authentication requests should return actionable, user-friendly error messages to guide users in successfully connecting their accounts.
```